client/pages/Assessment/pages/Section/components/DataTable/Table/Table.tsx

```diff
@@ -23,37 +25,47 @@ const Table: React.FC<Props> = (props) => {
   const { i18n } = useTranslation()
   const [printView] = [false] // usePrintView()
 
+  const countryIso = useCountryIso()
   const odp = table.props.odp === true
   const secondary = false // todo: missing table.props.secondary === true
   const rowsHeader = rows.filter((row) => row.props.type === RowType.header)
   const rowsData = rows.filter((row) => row.props.type !== RowType.header)
   const tableRef = useRef<HTMLTableElement>(null)
   const displayTableExportButton = !secondary && !printView && tableRef.current != null
 
+  if (!data) return null
```

Comment:
why not at the Datatable level as it was?

Reply:
typo, fixed